### ISSUE TITLE
[A11y] Desktop/Mobile Explore Collection - List: Column Headers

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1809]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Explore Collections

### Admin user?

No

### Issue Description

Explore Collection - List: https://staging.oregondigital.org/explore_collections/uo?locale=en

The table column headers are not being read via the screenreader.

### Suggested Fix

See Section for aria labels: WAI-ARIA Roles, States, and Properties
https://w3c.github.io/aria-practices/#table

**Comment by azle** [comment]
@CGillen Original Ticket is fixed, however when I ran it through the validator received 2 errors:

1 missing form label
<img width="1115" alt="Screen Shot 2021-12-13 at 9 14 30 AM" src="https://user-images.githubusercontent.com/8353860/145857711-c26aff94-305c-4212-b2e7-d76f85a5613a.png">

1 broken aria reference
<img width="967" alt="Screen Shot 2021-12-13 at 9 14 49 AM" src="https://user-images.githubusercontent.com/8353860/145857758-7db665b5-f85b-4c3f-b914-f2a3ca3d4d58.png">
